gdc_ng_models/models/released_data.py

```diff
@@ -78,6 +78,15 @@ def __repr__(self):
         server_default=release_data_log_id_seq.next_value(),
     )
     release_number = schema.Column(sqltypes.Text, nullable=False)
+    action = schema.Column(sqltypes.Text, nullable=False, default="release")
```

Comment:
I added a default here to allow our existing data-tools to continue to work.   Once the admin api work is done, we'll no longer need the data-tools.  Alternatively, we could remove the default that update the data-tools and data files to require the action column.

I'm open to either.  Do we have a preference?

Reply:
If we intend to keep the existing table the nullable False will probably not work since there are existing rows that would be null after ddl query is run. imo we shouldn't worry about trickling changes to data tools since we can always update it anytime.

Reply:
Good point about `NOT NULL`.  I assumed that the tables only existed in `qa-uat`, but I may be wrong about that.  They definitely do not exist in prod yet.   Not sure if they exist in `qa-orange` yet.  If they exist in `qa-orange`, we should just drop the table an re-populate it with data-tools, as that would be the same steps as we'd need for the prod deployment of VBF.

Reply:
Ended up removing the default value.  For our initial population of `released_data`, we are not populating `released_data_log` automatically.